packages/server/src/auth/auth.controller.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { LoginController, SignupController } from './auth.controller';
+import { AuthService } from './auth.service';
+
+describe('LoginController', () => {
+  let loginController: LoginController;
+
+  beforeEach(async () => {
+    const app: TestingModule = await Test.createTestingModule({
+      controllers: [LoginController],
+      providers: [AuthService]
+    }).compile();
+
+    loginController = app.get<LoginController>(LoginController);
+  });
+
+  describe('/signup', () => {
+    it('Should return JWT token', async () => {
+      const project_id = '1';
+      const username = 'username';
+      const method = 'username';
+      const password = 'pw';
+      const expectedResp = `${project_id}-${username}-${method}-${password}`;
+      await expect(loginController.login(project_id, username, password)).resolves.toBe(expectedResp);
+    });
+  });
+});
+
+
+describe('SignupController', () => {
+  let signupController: SignupController;
+
+  beforeEach(async () => {
+    const app: TestingModule = await Test.createTestingModule({
+      controllers: [SignupController],
+      providers: [AuthService]
```

Comment:
Shouldn't the `AuthService` be a spy with mocked methods?

Reply:
Removed these tests for now. Will increase coverage after PR.